examples/callbacks/config.yaml

```diff
@@ -0,0 +1,9 @@
+hydra:
+  callbacks:
+    example:
+      _target_: "example_callbacks.ExampleCallbacks"
+
+    order:
+      - example
```

Comment:
How about we just follow the order as specified by `callbacks` node and not add an extra node?

Reply:
yea, I think that makes more sense.